### PR TITLE
Update user-service logging script to include userId

### PR DESCRIPTION
**Pull request by donnyhyon**
User service outputs logs that identify which forces are using which domain to access Bichard service.

This PR modifies this script to also include the userId to allow us to identify which users are using the old url

### CODE REVIEW

**Review by emadkaramad — DISMISSED** [review]

File: src/components/StatsGathering/ForceBrowserShareAssets.tsx [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
+  userId: number | undefined
 }
 
-const ForceBrowserShareAssets = ({ visibleForces }: Props) => {
+const ForceBrowserShareAssets = ({ visibleForces, userId }: Props) => {
   const forces = visibleForces.split(",")
+  const id = userId?.toString()
   const forcesImgs = forces.map((force: string, i: number) => (
-    <img src={`/forces.png?forceID=${force}`} className="govuk-!-display-none" key={i} />
+    <img src={`/forces.png?forceID=${force}?userID${id}`} className="govuk-!-display-none" key={i} />
```
Comment (bjpirt) [inline src/components/StatsGathering/ForceBrowserShareAssets.tsx:12]: This needs a tweak - you've got two `?` and no `=`

**Review by bjpirt — CHANGES_REQUESTED** [review]
See comment

File: src/components/StatsGathering/ForceBrowserShareAssets.tsx (hunk 1 above)
Comment (donnyhyon) [reply src/components/StatsGathering/ForceBrowserShareAssets.tsx]: Thanks, updated to the below. 
```
<img src={`/forces.png?forceID=${force}&userID=${id}`} className="govuk-!-display-none" key={i} />
```

**Review by richardcane — DISMISSED** [review]

File: src/components/StatsGathering/ForceBrowserShareAssets.tsx [hunk 2]
```diff
@@ -2,12 +2,14 @@
 /* eslint-disable jsx-a11y/alt-text */
 interface Props {
   visibleForces: string
+  userId: number
 }
 
-const ForceBrowserShareAssets = ({ visibleForces }: Props) => {
+const ForceBrowserShareAssets = ({ visibleForces, userId }: Props) => {
   const forces = visibleForces.split(",")
+  const id = userId.toString()
```
Comment (bjpirt) [inline src/components/StatsGathering/ForceBrowserShareAssets.tsx:10]: I don't think you need to explicitly convert this; a) I think it's probably already a string and b) using it in a template string will automatically convert it

**Review by bjpirt — DISMISSED** [review]

**Review by bjpirt — APPROVED** [review]

**Pull request merged**